packages/fast-components-react-msft/src/radio/index.ts

```diff
@@ -0,0 +1,8 @@
+import * as React from "react";
+import {
+    Radio
+} from "@microsoft/fast-components-react-base";
+import manageJss, { IJSSManagerProps } from "@microsoft/fast-jss-manager-react";
+import { IDesignSystem, RadioStyles } from "@microsoft/fast-components-styles-msft";
+
+export default manageJss(RadioStyles)(Radio);
```

Comment:
just like with base you need to export this from the `/fast-comopnents-react-msft/src/index.ts`

Reply:
exported